src/variables.jl

```diff
@@ -96,15 +96,29 @@ variable_ref_type(v::AbstractVariableRef) = typeof(v)
 Holds a reference to the model and the corresponding MOI.VariableIndex.
 """
 struct VariableRef <: AbstractVariableRef
-    m::Model
-    index::MOIVAR
+    model::Model
+    index::MOI.VariableIndex
 end
 
-# Should be implemented by all `AbstractVariableRef`
-owner_model(v::VariableRef) = v.m
+# owner_model should be implemented by all `AbstractVariableRef`.
+"""
+    owner_model(v::VariableRef)
```

Comment:
This could be part of the AbstractVariableRef API

Reply:
What does that mean in practice? Moving this docstring to `function owner_model end`?

Reply:
Yes, and using `::AbstractVariableRef`

Reply:
Done